src/CoreBundle/Resources/views/snippets/CMSModuleChooser/headerjs.html.twig

```diff
@@ -2,6 +2,7 @@
 
 <link href="{{ themePath }}/css/cms_user_style/main.css" rel="stylesheet" type="text/css" />
 <link href="{{ resourcePath }}/css/modules/templateengineeditor.css" rel="stylesheet" type="text/css" />
+<link href="{{ resourcePath }}/iconFonts/fontawesome-free-5.8.1/css/all.css" media="screen" rel="stylesheet" type="text/css" />
```

Comment:
In Chameleon 6.3.0 the correct version is 5.5.0.

Reply:
fixed